File: pset1/mario.c
#include<stdio.h>
#include <cs50.h>

int main(void)
{
    int n ;
    do
    {
        printf("Height: ");
         n = GetInt();
         
    }while(n < 0 || n > 23);
    
    for(int i=1; i<=n; i++)
    {
        for(int j=1; j<=n-i; j++)
        {
            printf(" ");
        }
        for(int j=1; j<=i+1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
    
    
}

File: pset6/mario.py
import cs50


n = 0 ;
while True:
    print("Height: ",end='');
    n = cs50.get_int();
    if n>=0 and n<=23:
        break;

for i in range(1,n+1):
    for j in range(1,n-i+1):
        print(" ",end='');
    for j in range(1,i+2):
        print("#",end='');
    print();
    
    
    
